lib/improver/ensemble_calibration/ensemble_calibration_utilities.py

```diff
@@ -44,23 +44,21 @@ def convert_cube_data_to_2d(
     Function to convert data from a N-dimensional cube into a 2d
     numpy array. The result can be transposed, if required.
 
-    Parameters
-    ----------
-    forecast : Iris cube
-        N-dimensional cube to be reshaped.
-    coord : String
-        The data will be flattened along this coordinate.
-    transpose : Logical
-        If True, the resulting flattened data is transposed.
-        This will transpose a 2d array of the format [:, coord]
-        to [coord, :].
-        If False, the resulting flattened data is not transposed.
-        This will result in a 2d array of format [:, coord].
-
-    Returns
-    -------
-    forecast_data : Numpy array
-        Reshaped 2d array.
+    Args:
+        forecast (Iris cube):
+            N-dimensional cube to be reshaped.
+        coord (String):
```

Comment:
String should be string

This is repeated throughout this module. I shall not add a comment for every one.